pkg/chart/v2/util/jsonschema.go

```diff
@@ -31,13 +32,14 @@ import (
 func ValidateAgainstSchema(chrt *chart.Chart, values map[string]interface{}) error {
 	var sb strings.Builder
 	if chrt.Schema != nil {
+		slog.Debug("chart name", "chart-name", chrt.Name)
```

Comment:
I did a test of this and `chrt.Name` came out as `0x1031fed60`. This was not the chart name. This is a reference to it, I think. In any case, this needs to be fixed.

Reply:
Saw what I did here fixing it now.